### ISSUE TITLE
[Quest #3804] Army of the Damned

### ISSUE THREAD

**Issue by MightyFlix** [dalaranwow/dalaran-wow#5707]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Same problems described by OP of #3804 are happening again, despite the quest being marked as Fixed on Live Server
**Expected behaviour**: Soul Cleave (58913) ability doesn't work (no target error), Lordaeron units get bugged when attacking and reset position, raised ghouls do too little damage.

**Steps to reproduce the problem**:

1.  Take the quest
2. Try and attack Lordaeron units or kill them using Deathstorm (58912) and use Gift of the Lich King (31268) to raise ghouls from the corpses
3. Ghouls do almost no damage and you're not able to use Soul Cleave (58913) to get mana and hence complete the quest

**Include proofs for this behaviour**

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
ID on title

**Include Screenshots from the issue if necessary**

![WoWScrnShot_031819_172358](https://user-images.githubusercontent.com/33992970/54561250-f8d8d400-49a2-11e9-9b6b-768dd9890f0b.jpg)
![WoWScrnShot_031819_172416](https://user-images.githubusercontent.com/33992970/54561285-0beba400-49a3-11e9-93ef-3d477addb039.jpg)

**Comment by Ergates82** [comment]
Yes, I can confirm that this is still bugged as of yesterday, and no action seems to have been taken to fix the issue.
Only known way to complete this quest is turn a handful of footmen into ghouls/immediately kill ghouls to gain mana/repeat - now all your mana is spent - exit the vehicle/talk to kid/get new vehicle/repeat cycle until ghoul count is reached....

This turns what was one of the game's more interesting and enjoyable quests into a long, boring, tedious slog - but at least it's completable
